panda/src/movies/microphoneAudioDS.cxx

```diff
@@ -146,7 +146,7 @@ find_all_microphones_ds() {
         format.nBlockAlign = 2 * chan;
         format.wBitsPerSample = 16;
         format.cbSize = 0;
-        stat = waveInOpen(NULL, i, &format, NULL, NULL, WAVE_FORMAT_QUERY);
+        stat = waveInOpen(nullptr, i, &format, nullptr, nullptr, WAVE_FORMAT_QUERY);
```

Comment:
The fourth and fifth argument are `DWORD_PTR` which does not accept `nullptr` because it is an integral type.  Changing to `0` fixes the build.